python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -185,15 +185,35 @@ def build_run_status_sensor_context(
     )
 
 
+class JobAddress(
```

Comment:
What's the difference between `JobAddress` and `PipelineSelector` (maybe renamed to `JobSelector`)?

Reply:
ah didn't know `PipelineSelector` existed! they look pretty similar to me. I think this still needs a new type (PipelineSelector references solids) but I can make a `JobSelector` alongside `PipelineSelector`